docker/tpu/tpu_example.jsonnet

```diff
@@ -0,0 +1,25 @@
+local base = import 'templates/base.libsonnet';
+local tpus = import 'templates/tpus.libsonnet';
+
+local tputests = base.BaseTest {
+  frameworkPrefix: 'pl',
+  modelName: 'tpu-tests',
+  mode: 'presubmit',
+
+  timeout: 300, # 5 minutes, in seconds.
+
+  image: std.extVar('image'),
+  imageTag: std.extVar('image-tag'),
+
+  tpuSettings+: {
+    softwareVersion: 'pytorch-nightly',
+  },
+  accelerator: tpus.v3_8,
+
+  command: [
+    'py.test',
+    'pytorch-lightning/tests/models/test_tpu.py',
```

Comment:
```suggestion
    'pytorch-lightning/tests/models/test_tpu.py',
    '-v',
```

Reply:
I'll add this and some more fixes in a commit once the Github Secrets are set. I want to see if the Action runs into any more bugs than the current one (which is just that the Secrets are missing)